whenthen-web/src/components/AlertComponent/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { Overlay, Button } from 'react-bootstrap';
+import useRootData from '../../hooks/useRootData';
+import stylesDesktopDefault from './DesktopDefault.module.scss';
+
+const AlertComponent = () => {
+  const {
+    screenClass,
+    alertModalVisibility,
+    alertModalContent,
+    setAlert,
```

Comment:
remove it if you're not going to use it here

Reply:
setAlert method will be removed in the next commit.